api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -127,10 +128,9 @@ public void checkFreeTierBillingUsage() {
       updateFreeTierWorkspacesStatus(user, BillingStatus.INACTIVE);
     }
 
-    final Set<DbUser> usersWithNonNullRegistration =
-        userDao.findByFirstRegistrationCompletionTimeNotNull();
-    final Set<DbUser> usersToThresholdCheck =
-        Sets.difference(usersWithNonNullRegistration, expiredUsers);
+    final List<DbUser> allUsers = userDao.findUsers();
+    final Collection<DbUser> usersToThresholdCheck =
+        CollectionUtils.subtract(allUsers, expiredUsers);
```

Comment:
I see mixed of using Apache Common with Guava(Sets.difference above). Worth to settle on one util library? 
Also, not familiar with `CollectionUtils.subtract` what is the difference with `list.removeAll`?

Reply:
That might be a good idea.  I don't have a preference between those and I use whichever is most convenient.

I prefer these to `removeAll()` because they allow functional style instead of returning a boolean.